beetsplug/play.py

```diff
@@ -127,15 +127,10 @@ def play_music(self, lib, opts, args):
         self._log.debug('executing command: {} {}', command_str,
                         b' '.join(open_args))
         try:
-            util.interactive_open(open_args, command_str)
+            util.interactive_open(open_args, command_str, take_over=False)
```

Comment:
Perhaps it would be easier to just bypass `interactive_open` altogether and just do something like:

```py
util.command_output(util.shlex_split(command) + open_args)
```

In fact, beets' need for the `interactive_open` utility function is rapidly vanishing…